### ISSUE TITLE
Warning about not running xserver in headless mode with EGL

### ISSUE THREAD

**Issue by tjahn** [pyvista/pyvista#1278]
Hello,

I get a warning that the xserver is not running even if using VTK headless with EGL as described in docs https://dev.pyvista.org/extras/building_vtk.html.

```text
lib64/python3.6/site-packages/pyvista/plotting/plotting.py:93: UserWarning: 
This system does not appear to be running an xserver.
PyVista will likely segfault when rendering.

Try starting a virtual frame buffer with xvfb, or using
  ``pyvista.start_xvfb()``

  warnings.warn('\n'
```

**Comment by akaszynski** [comment]
That's something I didn't consider.  I'll have to check if EGL is installed before issuing the warning.

**Comment by akaszynski** [comment]
Fixed in #1294
